lib/utils.bash

```diff
@@ -892,7 +888,9 @@ remove_path_from_path() {
   # Output is a new string suitable for assignment to PATH
   local PATH=$1
   local path=$2
-  substitute "$PATH" "$path" "" | sed -e "s|::|:|g"
+  PATH=${PATH//"$path"}
```

Comment:
There is an argument to be made for using the substitute call everywhere to make it more easily understandable, but at the very least, here the sed pipe is unnecessary and this can simply be modified twice.